app/validators/routes_and_redirects_validator.rb

```diff
@@ -123,7 +123,6 @@ def initialize(record, redirect)
     def validate
       path = redirect[:path]
       destination = redirect[:destination]
-      type = redirect[:type]
```

Comment:
If this is something that was missing in previous PRs, I would add this commit to the first position explaining that we are fixing a previous issue and that all tests are passing in the commit notes.
If this is not the case, I would squash into proper commit.